sdcard/USR/SPLASH/README.txt

```diff
@@ -0,0 +1,15 @@
+Splash screen images files must be 24-bit BMP files of size 240x304 or 240x300 pixels.
+The current splash screen image is the one stored in the root directory of the SD card
```

Comment:
It's now stored in the /USR directory, not the root directory, correct?

Reply:
Thanks! fixed